no compile

```
me@gentoo ~/src/nxt-standalone $ make
[  1%] Generating files for libNXT.
Traceback (most recent call last):
  File "/home/me/src/nxt-standalone/generator/main.py", line 427, in <module>
    sys.exit(main())
  File "/home/me/src/nxt-standalone/generator/main.py", line 424, in main
    do_renders(renders, args.template_dir, args.output_dir)
  File "/home/me/src/nxt-standalone/generator/main.py", line 218, in do_renders
    output = env.get_template(render.template).render(**params)
  File "/usr/lib64/python3.4/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib64/python3.4/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib64/python3.4/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/me/src/nxt-standalone/generator/main.py", line 174, in get_source
    source = self.preprocess(f.read())
  File "/home/me/src/nxt-standalone/generator/main.py", line 193, in preprocess
    result.append(self.remove_indentation(line, indentation_level))
  File "/home/me/src/nxt-standalone/generator/main.py", line 201, in remove_indentation
    for _ in range(n):
TypeError: 'float' object cannot be interpreted as an integer
make[2]: *** [CMakeFiles/nxt.dir/build.make:65: generator/nxt/nxt.h] Error 1
make[1]: *** [CMakeFiles/Makefile2:67: CMakeFiles/nxt.dir/all] Error 2
make: *** [Makefile:128: all] Error 2
me@gentoo ~/src/nxt-standalone $ 
```
python  3.4, jinja 2.9.5

--- a/generator/main.py
+++ b/generator/main.py
@@ -1,3 +1,4 @@
+#!/usr/bin/env python2
 # Copyright 2017 The NXT Authors
 #
 # Licensed under the Apache License, Version 2.0 (the "License");

